docs/store-operations/tax/tax-properties.mdx

```diff
@@ -1,25 +1,27 @@
 ---
-title: Tax Properties for Products
-keywords: get all products, tax, tax properties, 
+title: Tax Properties
+keywords: get all products, tax, tax properties,
```

Comment:
A little bit funky having `get all products` here? 🤔 

Perhaps worth removing, I'm not sure I can justify a customer equivalent here.